pydevd_frame.py

```diff
@@ -396,6 +399,17 @@ def trace_dispatch(self, frame, event, arg):
                             finally:
                                 if val is not None:
                                     thread.additionalInfo.message = val
+
+                        if event == 'call':
```

Comment:
This check should only be done once right? (so, regardless on whether it matches the first time or not, a flag should be set at the debugger so that we don't do this check again)